docker/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.1"
+services:
+  run_lake_metrics:
+    image: jrossusgs/lakesout:v0.1
```

Comment:
Happy to use my personal docker hub for this as a stopgap, but long-term it might be better to have this on USGS's container registry, or a docker hub account that you control.

One model to consider longer-term is what the DRB inland salinity team did. They have their pipeline code on GitHub [here](https://github.com/USGS-R/drb-inland-salinity-ml). But they keep the container code on GitLab [here](https://github.com/USGS-R/drb-inland-salinity-ml). This allows them to build the container with CI, and to host it on a shared USGS resource.

Reply:
Awesome! I'd like to leave that kind of determination for best practice up to others since I am just barely a user of this container approach and don't know nearly enough to make decisions about it 😄